ajax/libs/noti.js/package.json

```diff
@@ -16,6 +16,6 @@
   "license": "GPL-2.0",
   "repository": {
     "type": "git",
-    "url": "https://github.com/j-l-n/noti.js.git"
+    "url": "https://github.com/needim/noty.git"
```

Comment:
The origin [git repo](https://github.com/cdnjs/cdnjs/tree/master/ajax/libs/noti.js) is not exist.
But I think the git repo which url is [https://github.com/needim/noty.git](https://github.com/needim/noty.git) , should be a new added.
Because the code of [this library](https://github.com/needim/noty) is different from [origin library](https://github.com/cdnjs/cdnjs/tree/master/ajax/libs/noti.js).